public_html/common.php

```diff
@@ -71,10 +71,12 @@ function load_wiki_name($query_array) {
 	$short_project = $project_map[$project];
 
 	// We set database name directly if received as a valid parameter
-	if ( !(empty($database)?) ) {
```

Comment:
Ah, I see! I didn't realize that was a ternary operator at the end here, which is why it wasn't working. I introduced 'none' to deal with that. I not used to thinking in php.